### PR TITLE
Upgrade kotlin version from 1.3.71 to 1.4.31

### PR DESCRIPTION
**Pull request by bbaldino**

### CODE REVIEW

**Comment by bbaldino** [comment]
I'll fix the `jtor` -> `ktor` typo in the commit when merging.

File: src/main/kotlin/org/jitsi/jibri/webhooks/v1/WebhookClient.kt [hunk 1]
```diff
@@ -83,6 +83,7 @@ class WebhookClient(
     }
 
     private val client = client.config {
+        expectSuccess = false
```
Comment (JonathanLennox) [inline src/main/kotlin/org/jitsi/jibri/webhooks/v1/WebhookClient.kt:86]: Is this related to the Kotlin change?

Comment (bbaldino) [reply src/main/kotlin/org/jitsi/jibri/webhooks/v1/WebhookClient.kt:86]: It was something I encountered from the ktor upgraded, I explained it in that commit message (easy to miss in github):
```
note that we now need to explicitly set "expectSuccess" to false to
disable the default response validator in the ktor client.  this was
always the desired behavior, but due to a bug
(https://youtrack.jetbrains.com/issue/KTOR-1412) the response validation
was not kicking in before.
```

**Review by JonathanLennox — APPROVED** [review]
